### DISCUSSION TITLE
Is there a way to pass configuration parameters on the command line?

### DISCUSSION THREAD

**Discussion by jjcaballero** [renovatebot/renovate#13681] [Request Help] [answered]
Is there a way to pass configuration parameters on the command line when calling renovate?
If so, is there anything in the documentation?
My goal would be to override the baseBranches parameter from the config file.
Thanks!

**Comment by rarkins** [comment, marked as answer]
Yes, you can use e.g. `--base-branches=main,dev`

**Reply by viceice** [reply]
https://docs.renovatebot.com/getting-started/running/#global-config

**Reply by viceice** [reply]
run `renovate --help` 🙃
